frontend/server/src/DAO/QualityNominations.php

```diff
@@ -25,51 +25,64 @@ public static function getNominationStatusForProblem(
         \OmegaUp\DAO\VO\Problems $problem,
         \OmegaUp\DAO\VO\Identities $identity
     ) : array {
-        $sql = '
-            SELECT
-                COUNT(r.run_id) > 0 as solved,
-                (SELECT
-                    COUNT(*)
+        $response = [
+            'nominated' => false,
+            'dismissed' => false,
+            'nominatedBeforeAC' => false,
+            'dismissedBeforeAC' => false,
+        ];
+
+        $sql = "SELECT
+                    qnn.contents
                 FROM
                     QualityNominations qnn
                 WHERE
-                    qnn.problem_id = p.problem_id AND
-                    qnn.user_id = i.user_id AND
-                    qnn.nomination = \'suggestion\'
-                ) as nominated,
-                (SELECT
-                    COUNT(*)
+                    qnn.problem_id = ? AND
+                    qnn.user_id = ? AND
+                    qnn.nomination = 'suggestion'
+                ORDER BY
+                    qnn.qualitynomination_id DESC";
+
+        /** @var array<string, string>|null $suggestion */
```

Comment:
```suggestion
        /** @var null|array{contents: string} $suggestion */
```

aquí es mejor enumerar las llaves que tiene el diccionario en vez de decir que es un mapa de cadenas->cadenas.

igual abajo en L74.